<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SNB-LAP" tests="4" failures="1" timestamp="10 Nov 2016 10:13:28 GMT" time="0.042" errors="0">
  <testcase name="test1" time="0.011" classname="Dev001"/>
  <testcase name="test2" time="0.003" classname="Dev001">
    <failure type="java.lang.AssertionError" message="Dev fail">
      <![CDATA[java.lang.AssertionError: Dev fail
	at Dev001.test2(Dev001.java:18)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.001" classname="Test001"/>
  <testcase name="test2" time="0.0" classname="Test001"/>
</testsuite>
